Add feature request workflow to contributing guidelines (#18528)
directus/directus: docs/contributing/pull-request-process.md

# Pull Request Process

Pull Requests (PRs) are a fantastic way to contribute back to a project. It's one of the fastest ways to see a bug fix
or new feature you care about land in the platform.

Reviewing and maintaining community submitted code is a very time consuming process. To ensure the core team can give
every PR the tender love and care it deserves, and not have valuable PRs to stale, we require that:

> **Every PR is required to fix/close an open [Issue](https://github.com/directus/directus/issues).**

Reviewer: ```suggestion
**Every pull request must be in answer to an existing open [Issue](https://github.com/directus/directus/issues).**
```

We use [GitHub Issues](https://github.com/directus/directus/issues) as a living to-do list of tasks we want to work on
next. Requiring PRs to implement Issues makes sure that your PR aligns with the core team's planning and long-term
goals. Please leave a comment on the issue you're preparing a PR for, so you can be assigned which makes sure nobody
else will work on the same thing at the same time.

> ## Choosing what to Implement

Reviewer 2: ```suggestion
## Choosing What to Implement
```
